nmtwizard/framework.py

```diff
@@ -842,29 +850,43 @@ def _get_vocab_info(self,
         if previous_vocab is not None:
             previous_vocab = self._convert_vocab(
                 previous_vocab, basename=previous_basename)
-            del opt['previous_vocabulary']
+            if 'previous_vocabulary' in opt:
+                del opt['previous_vocabulary']
```

Comment:
You could replace all `if x in y: del y[x]` with `y.pop(x, None)` to make the code more compact.